src/util/handlebars/handlebar-helpers.ts

```diff
@@ -27,6 +27,8 @@ export const registerHandlebarsHelpers = () => {
 				: `${numberOfSyncedRepos} / ${totalNumberOfRepos}`
 	);
 
+	hbs.registerHelper("checkRepoCount", (totalNumberOfRepos: unknown | undefined) => (typeof totalNumberOfRepos === "number" &&  totalNumberOfRepos >= 0));
```

Comment:
/nit can we use `number` instead of `unknown`? There may be handlebarsy reasons you did that so ok to ignore if thats the case

Reply:
yes @joshkay10 
this has been done because of handlebar reason